coopr-docs/docs/source/installation/upgrade.rst

```diff
@@ -0,0 +1,90 @@
+..
+   Copyright © 2015 Cask Data, Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+ 
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+.. index::
+   single: Coopr Upgrade
+
+==================
+Coopr Upgrade
+==================
+
+.. highlight:: console
+
+This guide describes how to upgrade Cask Coopr from 0.9.8 to the |release| Release.
+
+Overview
+========
+
+This upgrade involves three key steps:
+ * A database upgrade
+ * Changing the provisioner server URI port
+ * Running the upgrade script
+
+
+Compatibility Notes
+-------------------
+
+Provisioner plugins must now extend from Coopr::Plugin::Automator or Coopr::Plugin::Provider. 
+Any existing custom plugins need to be updated accordingly.
+
+
+Upgrade Steps
+=============
+
+On the Coopr servers:
+
+1. Stop all processes
+
+  .. parsed-literal::
```

Comment:
If you are not including anything that requires parsing, it's better to not use a parsed literal, as they don't get colored by Pygments correctly. Use a double-colon, either on the previous line or on al in by itself instead.
If you are using a parsed literal, a |$| at the start of the line will be converted to the correct class, and will be highlighted if you add a .. container:: highlight before the block:
see https://github.com/caskdata/coopr/pull/932/files#diff-9dfca3b17b98e055482a07cf6212af3d for example...if you care.